safedelete/config.py

```diff
@@ -0,0 +1,8 @@
+HARD_DELETE = 0
+SOFT_DELETE = 1
+SOFT_DELETE_CASCADE = 2
+HARD_DELETE_NOCASCADE = 3
+NO_DELETE = 4
+
+DELETED_INVISIBLE = 10
+DELETED_VISIBLE_BY_PK = 11
```

Comment:
These should still be accessible from `utils.py` until next release (people may be using it). Please add DeprecationWarning there (see `django.utils.deprecation`).